dlpy/utils.py

```diff
@@ -1110,60 +1151,122 @@ def _convert_coco(size, box, resize):
     return (x_min, y_min, x_max, y_max)
 
 
-def _convert_xml_annotation(filename, coord_type, resize, name_file = None):
-    # always use en locale since we use this locale to generate our internal txt files
+def _convert_xml_annotation(filename, coord_type, resize, task = 'object detection', name_file = None):
```

Comment:
I am not sure if we are dependent on opencv.

Reply:
Opencv is not listed in setup.py. I don't want to make it as a necessary package when installing dlpy.